main.tf

```diff
@@ -1,6 +1,7 @@
 locals {
   enable_private_endpoint = length(var.master_authorized_networks_config) == 0
-  uuid                    = "${var.cluster_name}-${random_uuid.id.result}"
+  # 'resource.name'  Must be a match of regex '(?:[a-z](?:[-a-z0-9]{0,61}[a-z0-9])?)'"."
+  uuid = substr("${var.cluster_name}-${random_string.id.result}", 0, 55)
```

Comment:
given its 32 + 16 + 1, when do we go over 55?

Reply:
i capped the id and the name using "friendly" numbers, 55 is the absolute max this can be. Ill remove the trim since we are validating both pieces